repositories.md

```diff
@@ -119,27 +119,27 @@ The automated process MAY use the repository tools from our [Reference Implement
 
 **Figure 2.** *A proposed configuration of roles on the Director repository.*
 
-Unlike the Image repository, the Director repository does not delegate images. Therefore, the Director repository SHOULD contain only the Root, Timestamp, Snapshot, and Targets roles, as illustrated in Figure 2. In the rest of this section, we will discuss how metadata for each of these roles are produced.
+Unlike the Image repository, the Director repository does not delegate images. Therefore, the Director repository SHOULD contain only the Root, Timestamp, Snapshot, and Targets roles, as illustrated in Figure 2. In the rest of this section, we will discuss how metadata for each of these roles is produced.
```

Comment:
If I'm gonna be really pedantic, "metadata" is plural, so "are" was correct. But did we decide to follow standard spoken usage instead? I don't really care either way.

Reply:
Yes. I think we let that one go.